### PR TITLE
Allow jpg output format

### PR DESCRIPTION
**Pull request by johnreaganmoore**
Not sure if this is correct, but I want to be able to grab an image from a video and I think I need to be able to put an image file extension.

### CODE REVIEW

**Comment by johnreaganmoore** [comment]
```{
    "timeline": {
        "tracks": [
            {
                "clips": [
                    {
                        "asset": {
                            "type": "video",
                            "src": "https://shotstack-assets.s3-ap-southeast-2.amazonaws.com/footage/skateboarding.mp4",
                            "trim": 2
                        },
                        "start": 0,
                        "length": 1
                    }
                ]
            }
        ]
    },
    "output": {
        "format": "jpg",
        "resolution": "hd"
    }
}
```
Support sent me that as an example of grabbing a screenshot, but I am not able to use jpg as an output format in the ruby sdk.

File: lib/shotstack/models/output.rb [hunk 1]
```diff
@@ -167,7 +167,7 @@ def list_invalid_properties
     # @return true if the model is valid
     def valid?
       return false if @format.nil?
-      format_validator = EnumAttributeValidator.new('String', ["mp4", "gif", "mp3"])
+      format_validator = EnumAttributeValidator.new('String', ["mp4", "gif", "mp3", "jpg"])
```
Comment (jeffski) [inline lib/shotstack/models/output.rb:170]: Please can you add to line 187 as well `validator = EnumAttributeValidator.new('String', ["mp4", "gif", "mp3"])`

Comment (jeffski) [inline lib/shotstack/models/output.rb:170]: Let me merge the changes and I'll add this in. I will need to update the version as well and deploy to rubygems.

**Comment by jeffski** [comment]
Thanks for the pull request. We will be releasing a big update in the next week or two with all image formats and support for a heap of new features. I can merge this for now though so you aren't blocked.

**Comment by jeffski** [comment]
Thanks again @johnreaganmoore. Changes are merged and the gem has been deployed to RubyGems: https://rubygems.org/gems/shotstack.

**Pull request merged**